pypesto/petab/importer.py

```diff
@@ -354,6 +373,8 @@ def create_objective(
             Whether to force-compile the model if not passed.
         **kwargs:
             Additional arguments passed on to the objective.
+            In case of ordinal measurements, inner_solver_options can optionally be passed here,
+            otherwise, those given to the importer constructor (or inner solver defaults) will be chosen.
```

Comment:
It refers to "no inner_solver_options provided". I agree, it's ambiguous.
Will remove "In case of ordinal measurements,", it's not needed.

Reply:
I think "In case of ordinal measurements" is still important as those solver options do not apply in the case of standard hierarchical estimation.

Reply:
Yeah true. I'll make it clear that's the case:

In case of ordinal measurements, Inner_solver_options can optionally be passed here.
If none are given, inner_solver_options given to the importer constructor
(or inner solver defaults) will be chosen.